lib/erb_lint/linters/hard_coded_string.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'better_html/tree/tag'
+
+module ERBLint
+  module Linters
+    # Checks for hardcoded strings. Useful if you want to ensure a string can be translated using i18n.
+    class HardCodedString < Linter
+      include LinterRegistry
+
+      class ConfigSchema < LinterConfig
+        property :present, accepts: [true, false], default: true, reader: :present?
```

Comment:
Is this copied from the new line linter? It doesn't seem to be used anywhere

Reply:
oh weird, I thought I changed that, might have been messed up with my git-fu.